.buildkite/commands/run-ui-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+TEST_NAME=$1
+DEVICE=$2
+IOS_VERSION=$3
+
+echo "Running $TEST_NAME on $DEVICE for iOS $IOS_VERSION"
+
+echo "--- 📦 Downloading Build Artifacts"
+buildkite-agent artifact download build-products.tar .
+tar -xf build-products.tar
+
+echo "--- :wrench: Fixing VM"
+brew install openjdk@11
+sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- 🧪 Testing"
+xcrun simctl list >> /dev/null
```

Comment:
Might be worth adding a comment for this one (I think I already saw a comment elsewhere, maybe in another PR or repo, explaining that this was kind of a hack to ensure the simulator would kick off or something?)

Reply:
This is tracked here: https://github.com/Automattic/buildkite-ci/issues/7 and I'm hoping to move this into the shared actions repo once I figure out the shape of it in the other apps, so with any luck it'll be replaced soon 😁

Reply:
Ah, yeah, [this is the other repo where I saw that comment](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/425/files#diff-9be69f7058741a227f07c6ddadd577559a213b90273c85498216c9f0e2607360R12-R13), which I can back-track from [this comment](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/425#discussion_r679601649) which you already reference in that buildkite-ci issue 👍  
The circle is complete ✔️ 🎉